packages/daf-did-comm/src/action-handler.ts

```diff
@@ -19,6 +19,7 @@ export interface ActionSendDIDComm extends Action {
     type: string
     body: any
   }
+  headers?: HeadersInit
```

Comment:
Where is the `HeadersInit` type defined?

Reply:
Hi!

https://github.com/node-fetch/node-fetch/blob/master/%40types/index.d.ts#L16

Reply:
Is from `node-fetch`. I guess it is a Node.js default type, that's why I didn't import it. Ts build is running ok

Reply:
I lean more toward declaring it as `headers: Record<string, string>` since that doesn't make it dependant on node-fetch.

What do you think?

Reply:
LGTM! Do I have to address the change?

Reply:
```suggestion
  headers?: Record<string, string>
```